src/kirby/components/icon/icon.component.scss

```diff
@@ -12,6 +10,12 @@
   }
 }
 
+.outline {
+  border: solid 3px currentColor;
+  padding: size('m');
+  border-radius: 50%;
```

Comment:
Consider using a large pixel value instead - percentage won't give you equally rounded corners unless the border-box is squared (same width/height)